packages/@aws-cdk/aws-events/lib/archive.ts

```diff
@@ -0,0 +1,71 @@
+import { Duration, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { IEventBus } from './event-bus';
+import { EventPattern } from './event-pattern';
+import { CfnArchive } from './events.generated';
+
+/**
+ * The event archive properties
+ */
+export interface ArchiveProps {
+  /**
+   * The name of the archive.
+   *
+   * @default - none
```

Comment:
`@default - Automatically generated` I assume?

Reply:
That's right. Going to change this.